Sources/StreamVideo/OpenApi/generated/Models/UserObject.swift

```diff
@@ -0,0 +1,124 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public final class UserObject: @unchecked Sendable, Codable, JSONEncodable, Hashable {
```

Comment:
Seems like we have both `UserResponse` and `UserObject` - why do we need both? Also this one doesn't have the chat stuff, which is good.

Reply:
yes, we don't use them for now - best to remove this altogether.